src/App/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ParkChecklist from '../ParkChecklist/ParkChecklist';
+import "./App.css"
+
+const App = () => {
+  const apiLink = 'https://developer.nps.gov/api/v1/parks?api_key=jR5uonh1B6R19iLhdweq7nHBs0uGgKgwzqn3BNfb';
+
```

Comment:
Is the API key out in the open just for testing purposes? Ultimately We will be using an endpoint supplied by the BE server anyway if I understand correctly.

Reply:
yeah its out right now just for testing purposes, once we get it more tuned to the end itll be hidden and the endpoint will indeed be supplied from the backend